syncer/syncer.go

```diff
@@ -1501,7 +1511,8 @@ func (s *Syncer) handleRowsEvent(ev *replication.RowsEvent, ec eventContext) err
 		}
 	}
 
-	ti, err := s.getTable(originSchema, originTable, schemaName, tableName, ec.parser2)
+	// TODO(csuzhangxc): check performance of `getTabel` from schema tracker.
+	ti, err := s.getTable(originSchema, originTable, schemaName, tableName)
```

Comment:
in _release-1.0_, we cache the table schema in memory directly, I'm not sure whether it's so fast for schema tracker.